housekeeper/store/cli.py

```diff
@@ -209,14 +209,34 @@ def delete_case(context, yes, case_name):
 @click.option('-f', '--force', is_flag=True,
               help='auto confirm and override sanity checks')
 @click.option('-d', '--date', help="date of the particular run")
-@click.argument('case_name')
+@click.option('-a', '--all', 'clean_all', is_flag=True, default=False, help="select all runs")
+@click.option('-b', '--before', help='runs before a date')
+@click.option('-a', '--after', help='runs after a date')
+@click.option('-u', '--untagged-only', is_flag=True, default=False, help='only remove assets without archive type')
```

Comment:
Naming the option `untagged_only` does make that all questions asked later, are negated. e.g. most if statements with `untagged_only` will be like this:
``` if not untagged_only:```.

So I wonder if we should change this to `tagged_only`.